### PR TITLE
Fork New Feedstock

### PR DESCRIPTION
**Pull request by Arishamays1**
Added `hatch-vcs` to AnacondaRecipes from condaforge.
- removed noarch
- updated about section

### CODE REVIEW

**Comment by ScottBailey** [comment]
I believe we are expecting the host section to include `setuptools` and `wheel`.

Additionally, there should be a `doc_url`.

Less important, but still used is some of our legacy stuff is `license_family`.

**Review by ScottBailey — CHANGES_REQUESTED** [review]
See comment.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -37,8 +39,10 @@ about:
     This provides a plugin for Hatch that uses your preferred version
     control system (like Git) to determine project versions.
   dev_url: https://github.com/ofek/hatch-vcs
+  doc_url: https://pypi.org/project/hatch-vcs/
   license: MIT
-  license_file: LICENSE.txt
```
Comment (ScottBailey) [inline recipe/meta.yaml:38]: The license file does seem to be `LICENSE.txt` as seen here:
https://github.com/ofek/hatch-vcs/blob/v0.2.0/LICENSE.txt

Comment (Arishamays1) [reply recipe/meta.yaml:38]: Done! Thanks!

**Review by ScottBailey — APPROVED** [review]
lgtm

**Pull request merged**